gmail/src/views/partner.ts

```diff
@@ -20,15 +20,14 @@ function onEnrichCompany(state: State) {
 }
 
 function onLogEmail(state: State) {
```

Comment:
Is it a duplicate of onLogEmailPartner ? Can't it be re-used ?

Reply:
We could, but it means that we will need to move this event handler in an other file, and currently all event handlers are in the file in which they are called

Reply:
That's ok then. Keep it like this :) .